network-api/networkapi/buyersguide/models.py

```diff
@@ -148,3 +154,125 @@ class Product(models.Model):
 
     def __str__(self):
         return str(self.name)
+
+
+class ProductVote(models.Model):
+    votes = models.IntegerField(
+        null=False,
```

Comment:
It's at [False by default](https://github.com/django/django/blob/stable/1.11.x/django/db/models/fields/__init__.py#L145-L151), you don't need to specify it.